packages/backend/src/remote/activitypub/renderer/undo.ts

```diff
@@ -3,9 +3,11 @@ import { ILocalUser, User } from '@/models/entities/user.js';
 
 export default (object: any, user: { id: User['id'] }) => {
```

Comment:
I think we could add a type here:

```suggestion
export default (object: IObject, user: { id: User['id'] }) => {
```

The type check in line 5 makes me unsure wether we should allow `null`, from a cursory review I could not find a place where `null` may be passed in, nor does it make sense to do so.

Reply:
- リファクタではないので今回そこを修正する予定はない
- IObjectはinboxに入ってくる方の型なので厳密には違う